ref/feature/mpJwt/examples.adoc

```diff
@@ -1,3 +1,5 @@
+== Examples
+
```

Comment:
Is the feature manager config required for this example? It's not usually included since the feature doc already has a line to add the feature. Is the current intro sentence needed?